code/datums/brain_damage/creepy_trauma.dm

```diff
@@ -17,90 +17,88 @@
 	else
 		out_of_view()
 
 /datum/brain_trauma/special/obsessed/proc/out_of_view()
-	time_spent_away += 20
-	if(time_spent_away > 1800) //3 minutes
+	time_spent_away += 2 SECONDS
+	if(time_spent_away > 3 MINUTES) //3 minutes
 		SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "creeping", /datum/mood_event/notcreepingsevere, obsession.name)
 	else
 		SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "creeping", /datum/mood_event/notcreeping, obsession.name)
 
 /datum/brain_trauma/special/obsessed/on_lose()
 	..()
-
-	UnregisterSignal(obsession.mind, COMSIG_MIND_CRYOED)
+	UnregisterSignal(obsession, COMSIG_MIND_CRYOED)
 	antagonist?.trauma = null
 	owner.mind.remove_antag_datum(/datum/antagonist/obsessed)
 
 /datum/brain_trauma/special/obsessed/on_hug(mob/living/hugger, mob/living/hugged)
-	if(hugged == obsession)
+	if(hugged == obsession.current)
 		obsession_hug_count++
 
 /datum/brain_trauma/special/obsessed/proc/on_obsession_cryoed()
 	SIGNAL_HANDLER
 
-	UnregisterSignal(obsession.mind, COMSIG_MIND_CRYOED)
+	UnregisterSignal(obsession, COMSIG_MIND_CRYOED)
 	var/message = "You get the feeling [obsession] is no longer within reach."
 	obsession = find_obsession()
 	if(!obsession)//we didn't find one
 		lose_text = "<span class='warning'>[message] The voices in your head fall silent.</span>"
 		qdel(src)
 		return
-	RegisterSignal(obsession.mind, COMSIG_MIND_CRYOED, PROC_REF(on_obsession_cryoed))
+	RegisterSignal(obsession, COMSIG_MIND_CRYOED, PROC_REF(on_obsession_cryoed))
 	to_chat(owner, "<span class='warning'>[message] The voices have a new task for you...</span>")
 	antagonist.objectives = list()
-	antagonist.forge_objectives(obsession.mind)
-	to_chat(owner, "<B>You don't know their connection, but The Voices compel you to stalk [obsession], forcing them into a state of constant paranoia.</B>")
+	antagonist.forge_objectives(obsession)
+	to_chat(owner, "<span class='bold'>You don't know their connection, but The Voices compel you to stalk [obsession.name], forcing them into a state of constant paranoia.</span>")
 	owner.mind.announce_objectives()
 
 /datum/brain_trauma/special/obsessed/proc/find_obsession()
 	var/chosen_victim
 	var/list/possible_targets = list()
 	var/list/viable_minds = list()
-	for(var/mob/Player in GLOB.player_list)//prevents crewmembers falling in love with nuke ops they never met, and other annoying hijinks
-		if(Player.mind && Player.stat != DEAD && !isnewplayer(Player) && !isbrain(Player) && Player.client && Player != owner && SSjob.GetJob(Player.mind.assigned_role))
-			viable_minds += Player.mind
-	for(var/datum/mind/possible_target in viable_minds)
-		if(possible_target != owner && ishuman(possible_target.current))
-			possible_targets += possible_target.current
-	if(possible_targets.len > 0)
-		chosen_victim = pick(possible_targets)
+	for(var/mob/living/carbon/human/potential_target in GLOB.carbon_list)
```

Comment:
carbon list? why not just use the player list? or just the crew list? same as traitor objectives

Reply:
i like micro-optimizations

Reply:
it doesn't optimize anything. There are more carbons than players.

Reply:
ah 🤦🏻‍♀️